Update Influx Config Options Section in Docs

zoesyc opened the pull request:
<!--

Thank you for sending a pull request! Here are some tips:

1. If this is your first time, please read our contribution guide at https://github.com/grafana/grafana/blob/main/CONTRIBUTING.md

2. Ensure you include and run the appropriate tests as part of your Pull Request.

3. In a new feature or configuration option, an update to the documentation is necessary. Everything related to the documentation is under the docs folder in the root of the repository.

4. If the Pull Request is a work in progress, make use of GitHub's "Draft PR" feature and mark it as such.

5. If you can not merge your Pull Request due to a merge conflict, Rebase it. This gets it in sync with the main branch.

6. Name your PR as "<FeatureArea>: Describe your change", e.g. Alerting: Prevent race condition. If it's a fix or feature relevant for the changelog describe the user impact in the title. The PR title is used to auto-generate the changelog for issues marked with the "add to changelog" label.

7. If your PR content should be added to the What's New document for the next major or minor release, add the **add to what's new** label to your PR. Note that you should add this label to the main PR that introduces the feature; do not add this label to smaller PRs for the feature.

-->

**What is this feature?**

[Add a brief description of what the feature or update does.]

Updates `InfluxDB Configuration Options` > 'Configuration Options` to reflect the changes we've made to the InfluxDB configuration page.  

[Add a description of the problem the feature is trying to solve.]

This docs update is for all users setting up an InfluxDB data source. This is a complex and nuanced space with multiple product versions, query languages, and authentication options. We've simplified the UI and these doc changes are meant to reflect the new UI.

[Add information on what kind of user the feature is for.]

N/A

<!--

- Automatically closes linked issue when the Pull Request is merged.

Usage: "Fixes #<issue number>", or "Fixes (paste link of issue)"

-->

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [x] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

zoesyc committed:
Update _index.md

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=108264) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=108264) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/grafana?pullRequest=108264) it.</sub>

zoesyc committed:
removed test change

zoesyc committed:
Update config options - url and authentication required settings description

zoesyc committed:
Updated config options - url and authentication optional settings

zoesyc committed:
Update Name, Database Settings, and PDC sections

- add Name and DB settings screenshot
- removes language specific configuration instructions
- updates DB settings section, required and optional settings
- edit language in PDC section

zoesyc committed:
Minor edits to language

zoesyc committed:
Update settings x language table to use checks, formatting on DBRP mapping warning

zoesyc committed:
change checkmark style

zoesyc committed:
minor formatting changes

alyssajoyner commented on docs/sources/datasources/influxdb/configure-influxdb-data-source/_index.md:
```diff
@@ -60,97 +56,109 @@ Complete the following steps to set up a new InfluxDB data source:
+- **Allowed cookies** - Defines which cookies are forwarded to the data source. All other cookies are deleted by default.
+- **Timeout** - Set an HTTP request timeout in seconds.
 
+##### Custom HTTP Headers
```
Any header of this size gets a bit lost and is smaller than the text below. Could we consider sizing up headers of this size so they're more noticeable?

alyssajoyner committed:
Lint

zoesyc committed:
replace level 5 headers with bolded text

alyssajoyner committed:
Merge branch 'main' into zoesyc-influx-config-docs

lwandz13 reviewed (approved):
Great work! Thank you!

alyssajoyner committed:
Merge branch 'main' into zoesyc-influx-config-docs
